.gitlab-ci.yml

```diff
@@ -224,7 +224,20 @@ algolia_sync_preview:manual:
   allow_failure: true
   interruptible: true
   timeout: 2h
+  cache:
+    - *hugo_cache
+    - *yarn_cache
+    - *pip_cache
+  dependencies:
+    - build_preview
   script:
+    # build the site to get algolia.json ondemand
```

Comment:
@davidejones could we have `algolia.json` output as an artifact from the `build_preview` job and somehow use that here instead of building the site again?